src/config/constants/types.ts

```diff
@@ -47,6 +47,13 @@ export interface Address {
   56: string
 }
 
+export interface Token {
+  symbol: string
+  address?: Address
+  decimals?: number
```

Comment:
decimals shouldn't be optional, right?

Reply:
When we use bnb, it doesn't make sense to have a decimal